packaging/language/npm.py

```diff
@@ -253,7 +253,10 @@ def main():
     elif state == 'latest':
         installed, missing = npm.list()
         outdated = npm.list_outdated()
-        if len(missing) or len(outdated):
+        if len(missing):
+            changed = True
+            npm.install() 
+        elif len(outdated):
             changed = True
```

Comment:
Should this actually just be a separate if (not an elif)? If both lists are non-empty, you'll only do the missing ones...